tokio/src/macros/try_join.rs

```diff
@@ -30,6 +30,25 @@ macro_rules! doc {
         ///
         /// [`tokio::spawn`]: crate::spawn
         ///
+        /// # Fairness
```

Comment:
Please make sure that the section names for `try_join!` match those we have for `join!`.